tests/ethpm/backends/test_ipfs_backends.py

```diff
@@ -0,0 +1,50 @@
+import json
+import pytest
+
+from eth_utils import to_text
+
+from ethpm.backends.ipfs import (
+    DummyIPFSBackend,
+    InfuraIPFSBackend,
+    IPFSGatewayBackend,
+)
+
+
```

Comment:
Not quite sure how to test `IPFSGatewayBackend` and `InfuraIPFSBackend` without making HTTP requests.

Reply:
Not sure what's available in Python land but maybe there's a lib where you can mock responses so that if a request is made to `http://somewhere.com/some/thing` you can intercept and send a response accordingly.

Reply:
These are really integration style tests and their value seems questionable.  Maybe just 1-2 smoke tests (maybe using mocks) would be sufficient to know that everything is *wired up* correctly while still keeping the amount of mocking to a minimum.